### PR TITLE
Updated the default values for Python 3.10 and CUDA to 11.4.

### PR DESCRIPTION
**Pull request by ArchanaShinde1**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Comment by cdeepali** [comment]
@ArchanaShinde1, looks like test results and tmp453_skwm got added mistakenly. Please remove them.

**Review by alexlang74 — APPROVED** [review]

**Comment by cdeepali** [comment]
Added quotes for python version in github actions to get successful run. - https://github.com/actions/runner/issues/2015#issuecomment-1193375052.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </p>
 
 [![Installation Options](https://img.shields.io/badge/Install%20with-conda%20%7C%20pip-brightgreen)](#installing-the-open-ce-build-tools)
-[![Python Support](https://img.shields.io/badge/python-3.8%20%7C%203.9-blue.svg)](#requirements)
+[![Python Support](https://img.shields.io/badge/python-3.8%20%7C%203.9%7C3.10-blue.svg)](#requirements)
```
Comment (cdeepali) [inline README.md:8]:
```suggestion
[![Python Support](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C3.10-blue.svg)](#requirements)
```

File: README.md [hunk 2]
```diff
@@ -107,11 +107,11 @@ The following commands will build version 1.1.4 of the open-ce environment file
 open-ce build env --git_tag open-ce-v1.1.4 opence-env
 ```
 
-The following commands will use the `opence-env.yaml` Open-CE environment file from a specific Open-CE release to build all of the Open-CE packages for Python 3.8 and 3.9, including only CUDA builds. The commands should be run from within the same directory that contains `local_files`.
+The following commands will use the `opence-env.yaml` Open-CE environment file from a specific Open-CE release to build all of the Open-CE packages for Python 3.8 ,3.9 and 3.10, including only CUDA builds. The commands should be run from within the same directory that contains `local_files`.
```
Comment (cdeepali) [inline README.md:110]:
```suggestion
The following commands will use the `opence-env.yaml` Open-CE environment file from a specific Open-CE release to build all of the Open-CE packages for Python 3.8 , 3.9 and 3.10, including only CUDA builds. The commands should be run from within the same directory that contains `local_files`.
```

**Review by cdeepali — APPROVED** [review]

**Pull request merged**